nano/lib/work.cpp

```diff
@@ -145,6 +145,50 @@ uint64_t nano::work_v1::value (nano::root const & root_a, uint64_t work_a)
 }
 #endif
 
+double nano::normalized_multiplier (double const multiplier_a, uint64_t const threshold_a)
+{
+	static nano::network_constants network_constants;
+	debug_assert (multiplier_a >= 1);
+	auto multiplier (multiplier_a);
+	/* Normalization rules
+	// ratio = multiplier of max threshold with base threshold
+	// normalized = (multiplier + (ratio - 1)) / ratio;
+	Epoch 1
```

Comment:
Since the numbers below assume a ratio, this comment could include what the ratio is.

Reply:
Updated